README.md

```diff
@@ -8,6 +8,29 @@ This module allows to test the performance and response time of individual endpo
 ## Installation
 
 ```
-composer require spryker-sdk/benchmark
+composer require --dev spryker-sdk/benchmark
 ```
 
+
+## Usage
+
+### How to use in Spryker projects
+Make sure you include the benchmark as `require-dev` dependency:
+```
+composer require --dev spryker-sdk/benchmark
+```
+
+The Development bundle provides a convenience command:
+```
+console benchmark:run
```

Comment:
why is this not in the module itself? the development module shouldnt provide a "optional" command for a require-dev module of another module.

Reply:
Please clarify. I don't quite understand what is meant. This command refers to the module

Reply:
"The Development bundle provides a convenience command"

I would expect this module to do that.

Reply:
oh, I understand you. Done)